==20182== Memcheck, a memory error detector.
==20182== Copyright (C) 2002-2007, and GNU GPL'd, by Julian Seward et al.
==20182== Using LibVEX rev 1804, a library for dynamic binary translation.
==20182== Copyright (C) 2004-2007, and GNU GPL'd, by OpenWorks LLP.
==20182== Using valgrind-3.3.0-Debian, a dynamic binary instrumentation framework.
==20182== Copyright (C) 2000-2007, and GNU GPL'd, by Julian Seward et al.
==20182== For more details, rerun with: -v
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015EE4: (within /lib/ld-2.7.so)
==20182==    by 0x400AB93: (within /lib/ld-2.7.so)
==20182==    by 0x40061E4: (within /lib/ld-2.7.so)
==20182==    by 0x4008677: (within /lib/ld-2.7.so)
==20182==    by 0x400BE0C: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x400C4D0: (within /lib/ld-2.7.so)
==20182==    by 0x40120A8: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x90D1F7F: (within /lib/libc-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==  Address 0xc6f8bd0 is 16 bytes inside a block of size 17 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x4008B75: (within /lib/ld-2.7.so)
==20182==    by 0x400BE0C: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x400C4D0: (within /lib/ld-2.7.so)
==20182==    by 0x40120A8: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x90D1F7F: (within /lib/libc-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x90D20E6: __libc_dlopen_mode (in /lib/libc-2.7.so)
==20182==    by 0x90AC03C: __nss_lookup_function (in /lib/libc-2.7.so)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015D6E: (within /lib/ld-2.7.so)
==20182==    by 0x4011F31: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x67B7F01: gtk_theme_engine_get (in /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==  Address 0xc7e4c50 is 48 bytes inside a block of size 49 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x67B7F01: gtk_theme_engine_get (in /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x674026C: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742301: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742E28: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6743295: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x67433F2: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015D6E: (within /lib/ld-2.7.so)
==20182==    by 0x400780A: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==  Address 0xc7e4c50 is 48 bytes inside a block of size 49 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x67B7F01: gtk_theme_engine_get (in /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x674026C: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742301: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742E28: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6743295: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x67433F2: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015EE4: (within /lib/ld-2.7.so)
==20182==    by 0x4007817: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==  Address 0xc7e4c50 is 48 bytes inside a block of size 49 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x67B7F01: gtk_theme_engine_get (in /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x674026C: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742301: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742E28: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6743295: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x67433F2: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015EE4: (within /lib/ld-2.7.so)
==20182==    by 0x400A99D: (within /lib/ld-2.7.so)
==20182==    by 0x40061E4: (within /lib/ld-2.7.so)
==20182==    by 0x4008677: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==  Address 0xc7e4c50 is 48 bytes inside a block of size 49 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x67B7F01: gtk_theme_engine_get (in /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x674026C: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742301: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6742E28: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x6743295: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==    by 0x67433F2: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015EE4: (within /lib/ld-2.7.so)
==20182==    by 0x400DEE6: (within /lib/ld-2.7.so)
==20182==    by 0x400E093: (within /lib/ld-2.7.so)
==20182==    by 0x400A547: (within /lib/ld-2.7.so)
==20182==    by 0x90D22F3: (within /lib/libc-2.7.so)
==20182==    by 0x80A10F3: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A10A9: dlsym (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E571: g_module_symbol (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x7E9EC7E: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182==  Address 0xc7e4d08 is 48 bytes inside a block of size 49 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x4007823: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x67B7FFA: (within /usr/lib/libgtk-x11-2.0.so.0.1200.9)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015D33: (within /lib/ld-2.7.so)
==20182==    by 0x4011F31: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x77BC1C8: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==  Address 0xd97fc58 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x77BC1C8: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC288: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC306: pango_map_get_engines (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF459: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF6CE: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BFD3B: pango_itemize_with_base_dir (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015D33: (within /lib/ld-2.7.so)
==20182==    by 0x400780A: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==  Address 0xd97fc58 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x77BC1C8: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC288: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC306: pango_map_get_engines (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF459: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF6CE: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BFD3B: pango_itemize_with_base_dir (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015ECA: (within /lib/ld-2.7.so)
==20182==    by 0x4007817: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==  Address 0xd97fc58 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x77BC1C8: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC288: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC306: pango_map_get_engines (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF459: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF6CE: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BFD3B: pango_itemize_with_base_dir (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015ECA: (within /lib/ld-2.7.so)
==20182==    by 0x400A99D: (within /lib/ld-2.7.so)
==20182==    by 0x40061E4: (within /lib/ld-2.7.so)
==20182==    by 0x4008677: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==  Address 0xd97fc58 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x82E2EBB: g_malloc (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x82FA0EE: g_strdup (in /usr/lib/libglib-2.0.so.0.1600.4)
==20182==    by 0x7E9E932: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x7C85A2B: g_type_module_use (in /usr/lib/libgobject-2.0.so.0.1600.4)
==20182==    by 0x77BC1C8: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC288: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BC306: pango_map_get_engines (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF459: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BF6CE: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==    by 0x77BFD3B: pango_itemize_with_base_dir (in /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015ECA: (within /lib/ld-2.7.so)
==20182==    by 0x400AB93: (within /lib/ld-2.7.so)
==20182==    by 0x40061E4: (within /lib/ld-2.7.so)
==20182==    by 0x4008677: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==  Address 0xdc10bc8 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x4007823: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== Invalid read of size 8
==20182==    at 0x4015ECA: (within /lib/ld-2.7.so)
==20182==    by 0x400DEE6: (within /lib/ld-2.7.so)
==20182==    by 0x400E093: (within /lib/ld-2.7.so)
==20182==    by 0x400A547: (within /lib/ld-2.7.so)
==20182==    by 0x90D22F3: (within /lib/libc-2.7.so)
==20182==    by 0x80A10F3: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A10A9: dlsym (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E571: g_module_symbol (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x7E9EBF1: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182==  Address 0xdc10bc8 is 40 bytes inside a block of size 47 alloc'd
==20182==    at 0x4C22FAB: malloc (vg_replace_malloc.c:207)
==20182==    by 0x4007823: (within /lib/ld-2.7.so)
==20182==    by 0x40085CE: (within /lib/ld-2.7.so)
==20182==    by 0x4012048: (within /lib/ld-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x401191A: (within /lib/ld-2.7.so)
==20182==    by 0x80A0F8A: (within /lib/libdl-2.7.so)
==20182==    by 0x400DDF5: (within /lib/ld-2.7.so)
==20182==    by 0x80A14EC: (within /lib/libdl-2.7.so)
==20182==    by 0x80A0EF0: dlopen (in /lib/libdl-2.7.so)
==20182==    by 0x7E9E975: g_module_open (in /usr/lib/libgmodule-2.0.so.0.1600.4)
==20182==    by 0x77BC0DA: (within /usr/lib/libpango-1.0.so.0.2002.3)
==20182== 
==20182== ERROR SUMMARY: 16 errors from 12 contexts (suppressed: 19 from 1)
==20182== malloc/free: in use at exit: 7,030,461 bytes in 79,590 blocks.
==20182== malloc/free: 2,800,757 allocs, 2,721,167 frees, 314,507,800 bytes allocated.
==20182== For counts of detected errors, rerun with: -v
==20182== searching for pointers to 79,590 not-freed blocks.
==20182== checked 7,195,680 bytes.
==20182== 
==20182== LEAK SUMMARY:
==20182==    definitely lost: 606,845 bytes in 4,359 blocks.
==20182==      possibly lost: 113,560 bytes in 104 blocks.
==20182==    still reachable: 6,310,056 bytes in 75,127 blocks.
==20182==         suppressed: 0 bytes in 0 blocks.
==20182== Rerun with --leak-check=full to see details of leaked memory.
